src/components/Home.js

```diff
@@ -11,22 +12,53 @@ const Home = () => {
     url += '&app_id=' + ID;
     url += '&app_key=' + KEY;
 
+    const [recipes, setRecipes] = useState([]);
+    const [recipeTitles, setRecipeTitles] = useState([]);
+    const [recipeImages, setRecipeImages] = useState([]);
+    const [recipeIngredients, setRecipeIngredients] = useState([]);
+    const [recipeLinks, setRecipeLinks] = useState([]);
+    const [recipeIndexes, setRecipeIndexes] = useState([]);
+    const [cards, setCards] = useState(false);
+
     const getRandomRecipes = async () => {
         const response = await fetch(url);
         const body = await response.json();
-        console.log(body);
+        const recipeData = body.hits.map(hit => hit.recipe);
```

Comment:
This line is throwing an error if the user is unauthorized (no valid API keys). Up to you if you want to handle this in the code (since ideally in production this should never happen but it could)

Reply:
I have resolved this and handled the error.